client/my-sites/plan-features-list/index.jsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PlanFeatures from 'my-sites/plan-features';
+import { PLAN_FREE, PLAN_PREMIUM, PLAN_BUSINESS } from 'lib/plans/constants';
+import FAQ from 'components/faq';
+import FAQItem from 'components/faq/faq-item';
+
+class PlanFeaturesList extends Component {
+
+	render() {
+		const { translate, site } = this.props;
+
+		return (
+			<div>
+				<div className="plan-features-list">
+					<PlanFeatures plan={ PLAN_FREE } /* onClick={ this.upgradePlan } */ />
+					<PlanFeatures plan={ PLAN_PREMIUM } /* onClick={ this.upgradePlan } */ />
+					<PlanFeatures plan={ PLAN_BUSINESS } /* onClick={ this.upgradePlan } */ />
```

Comment:
This is great for now. :) We'll also need to be able to handle Jetpack plans, and personal plan display in follow up PRs.